components/ILIAS/Mail/classes/Mime/Transport/class.ilMailMimeTransportBase.php

```diff
@@ -90,11 +90,10 @@ final public function send(ilMimeMail $mail): bool
 
         $this->getMailer()->Subject = $mail->getSubject();
 
-        if ($mail->getFrom()->hasReplyToAddress() && !$this->getMailer()->addReplyTo(
-            $mail->getFrom()->getReplyToAddress(),
-            $mail->getFrom()->getReplyToName()
-        )) {
-            ilLoggerFactory::getLogger('mail')->warning($this->getMailer()->ErrorInfo);
+        if ($mail->getFrom()->hasReplyToAddress()) {
+            if (!$this->getMailer()->addReplyTo($mail->getFrom()->getReplyToAddress(), $mail->getFrom()->getReplyToName())) {
+                ilLoggerFactory::getLogger('mail')->warning($this->getMailer()->ErrorInfo);
+            }
```

Comment:
We don't need these changes for the feature, right? We should keep the PR minimal.